src/app/collective-rewards/active-builders/ActiveBuildersContent.tsx

```diff
@@ -20,6 +23,7 @@ const filterFunction = (builder: Builder, status: string) => {
 
 export const ActiveBuildersContent = () => {
   const { data: builders, isLoading, error } = useGetBuildersByState(undefined, true)
+  const filteredBuilders = builders?.filter(isDeactivated)
```

Comment:
I may move this to the `getMostAdvancedProposal` but it depends if product would like to show it as deactivated or not.

Reply:
We agree to remove it from the list